src/main/java/com/dev/cinema/controller/UserController.java

```diff
@@ -28,15 +30,19 @@ public UserController(UserService userService,
     }
 
     @GetMapping("/by-email")
-    public UserResponseDto findByEmail(@RequestParam String email) {
+    public UserResponseDto findByEmail(Authentication authentication) {
```

Comment:
please see checklist - we should not change this method

Reply:
done